prod.Dockerfile

```diff
@@ -0,0 +1,46 @@
+# Install dependencies only when needed
```

Comment:
hoping we don't need another docker image. set the ENV as a build arg.

Reply:
wops, thanks for the heads up. removed duplicate